express-get-json/server.ts

```diff
@@ -0,0 +1,40 @@
+import express from 'express';
+
+type Grades = {
+  [key: number]: { id: number; name: string; course: string; score: number };
+};
```

Comment:
This works. Personally, I would have written it as:
```
type Grade = { id: number; name: string; course: string; score: number };
type Grades = Record<number, Grade>;
```
(But if you're not comfortable with generic types yet, you can also write it as `type Grades = { [key: number]: Grade };`